index.js

```diff
@@ -16,6 +16,8 @@ const {
 
 const { DOMPoint, DOMMatrix, DOMRect } = require('./geometry')
 
+const loadImage = require('./loadImage')
```

Comment:
please rename the `loadImage.js` to `load-image.js`, we are using **kebab-case** style filename except for the test snapshot files.